aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -8,41 +8,33 @@ By [Peter Kellner](http://peterkellner.net)
 
 The ```AnchorTagHelper``` enhances the html anchor (`<a ... ></a>`) tag. A new set of attributes are defined that work with the anchor tag. The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL. That URL can include an optional protocol such as https.
```

Comment:
Yes, that was my point exactly. For my comment here, I may have assumed incorrectly that the reader doesn't know much (or anything) about TH's. That might be the wrong assumption. I can say ... I think this is fair ... that if the reader were to land on this doc and read that sentence  ...

>The link generated (on the href tag) is based on a combination of these new attributes that work together to form the final redirect URL.

... that they really wouldn't be able to understand what it means.

Whatever the case with all of that, I would remove the word "redirect" there, since that has special meaning.